content/docs/user-guide/settings/az-console-how-to-use-settings-registry.md

```diff
@@ -0,0 +1,173 @@
+---
+description: Describes how to execute AZ Console Commands using the Settings Registry
+title: Using AZ::Conosle to access the Settings Registry
+linktitle: Using AZ::Conosle to access the Settings Registry
+weight: 400
+---
+
+Preamble
+========
+
+The AZ::Console has been extended with support for the Settings Registry to allow invoking console commands via setting a JSON value underneath a specific key in the Settings Registry.
+The AZ::Console hooks into the Settings Registry Notification System as well as the JSON Patching Reporting system in order to detect changes to Settings Registry keys underneath the JSON object of "/Amazon/AzCore/Runtime/ConsoleCommands" and "/O3DE/Autoexec/ConsoleCommands"
+Where to place a console command depends on whether the user desires the console command should be merged to the bootstrap.game.\<config>.\<platform>.setreg file and therefore in the GameLauncher/ServerLauncher
```

Comment:
```suggestion
Where to place a console command depends on if the user wants the console command to be merged into the bootstrap.game.\<config>.\<platform>.setreg file and how it should be accessed by the GameLauncher/ServerLauncher.
```

unsure about `and therefore in the GameLauncher/ServerLauncher` as it seems like a trailing sentence and nothing below talks about the launchers specifically.

Reply:
Yeah this should explain that adding console commands to a file with the scheme of "<filename>.clients.setreg" will execute only in the GameLauncher(Should be called ClientLauncher, but whatever).
Having a file of "<filename>.servers.setreg" will execute only in the ServerLauncher.

Confusingly having a file called "<filename>.game.setreg" will execute in both the **Game**Launcher and ServerLauncher.

If GameLauncher was called ClientLauncher it wouldn't be as confusing.